config/openhpc.yml

```diff
@@ -72,6 +72,75 @@ cluster_roles:
 openhpc_slurm_partitions: 
   - "{{ slurm_compute }}"
 
+# Choose between glusterfs and beegfs
```

Comment:
Isn't it a valid choice not to have either of these, and only use CephFS via Manila?

Reply:
Maybe we need a `none.yml` noop playbook. That would probably do for now.

A future alternative would be to drive this using group membership. Could provide flags in here to enable gluster/beegfs, then build up the `cluster_roles` list based on these flags.

Reply:
what about `when: cluster_fs is defined`?

Reply:
oh, just realised that conditionals aren't allowed for importing playbook... not sure why...

Reply:
ive added a fake playbook called `none` instead as mark suggested